src/figuresClasses.ts

```diff
@@ -1,19 +1,84 @@
+
+enum Shape {
+  Triangle = 'triangle',
+  Rectangle = 'rectangle',
+  Circle = 'circle',
+}
+
 export interface Figure {
+  shape: Shape;
+  color: 'red' | 'green' | 'blue',
 
+  getArea: () => number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape = Shape.Triangle;
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('wrong length specified');
+    }
+
+    if ((a + b) <= c || (b + c) <= a || (c + a) <= b) {
+      throw new Error('wrong length specified');
+    }
+  }
 
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+
+    return Math.round(area * 100) / 100;
+  }
 }
 
-export class Circle {
+export class Circle implements Figure {
+  public shape = Shape.Circle;
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public radius: number,
+  ) {
+    if (radius <= 0) {
+      throw new Error('wrong length specified');
+    }
+  }
+
+  getArea(): number {
+    const area = Math.PI * this.radius ** 2;
 
+    return Math.trunc(area * 100) / 100;
```

Comment:
Create function for this `Math.trunc(... * 100) / 100;` and reuse in all places where you need to round your values